### DISCUSSION TITLE
使用tensor_parallelize训练bloom，仅保存了分片之后的参数

### DISCUSSION THREAD

**Discussion by yangjianxin1** [hpcaitech/ColossalAI#3247] [Community | Q&A]
使用两张gpu训练bloom，并且使用tensor_parallelize策略，调用colossalai.utils.save_checkpoint保存checkpoint时，仅保存了被切分之后一半的模型权重，无法保存完成的模型权重。

**Comment by taishiciR** [comment]
hi~@yangjianxin1 请问你在启用tensor_parallelize策略时，有同时用到lora配置吗，不知道lora是否支持tensor并行训练？

**Reply by yangjianxin1** [reply]
> hi~@yangjianxin1 请问你在启用tensor_parallelize策略时，有同时用到lora配置吗，不知道lora是否支持tensor并行训练？

没有使用Lora

**Comment by binmakeswell** [comment]
Hi @yangjianxin1 @taishiciR We will update this part next week. https://github.com/hpcaitech/ColossalAI/discussions/3339 Thanks.
